### PR TITLE
docs: list IE 9 and 10 as deprecated

### PR DESCRIPTION
**Pull request by atscott**
Add documentation in the deprecations markdown file about the deprecation of IE 9 and 10.

### CODE REVIEW

File: aio/content/guide/deprecations.md [hunk 1]
```diff
@@ -651,3 +652,18 @@ For more information about using `@angular/common/http`, see the [HttpClient gui
 | --------------------- | ------------------------------------------- |
 | `MockBackend` | [`HttpTestingController`](/api/common/http/testing/HttpTestingController) |
 | `MockConnection` | [`HttpTestingController`](/api/common/http/testing/HttpTestingController) |
+
+{a browser-support}
```
Comment (gkalpak) [inline aio/content/guide/deprecations.md:656]:
```suggestion
{@a browser-support}
```

File: aio/content/guide/deprecations.md [hunk 2]
```diff
@@ -651,3 +652,18 @@ For more information about using `@angular/common/http`, see the [HttpClient gui
...
+## Browser support
+
+{a ie-9-10}
```
Comment (gkalpak) [inline aio/content/guide/deprecations.md:659]:
```suggestion
{@a ie-9-10}
```

**Review by gkalpak — COMMENTED** [review]
I think it might be useful to add a note in [browser-support.md][1] as well.

[1]: https://github.com/angular/angular/blob/0b0692b513/aio/content/guide/browser-support.md

**Review by kara — CHANGES_REQUESTED** [review]
Thanks!

**Review by gkalpak — APPROVED** [review]

File: aio/content/guide/deprecations.md [hunk 3]
```diff
@@ -651,3 +652,18 @@ For more information about using `@angular/common/http`, see the [HttpClient gui
 | --------------------- | ------------------------------------------- |
 | `MockBackend` | [`HttpTestingController`](/api/common/http/testing/HttpTestingController) |
 | `MockConnection` | [`HttpTestingController`](/api/common/http/testing/HttpTestingController) |
+
+{@a browser-support}
+## Browser support
```
[the remark this answers is not in the document]
Comment (kara) [reply aio/content/guide/deprecations.md:657]: Looks like this comment still needs to be addressed?

File: aio/content/guide/deprecations.md [hunk 4]
```diff
@@ -651,3 +652,18 @@ For more information about using `@angular/common/http`, see the [HttpClient gui
...
+
+{@a ie-9-10}
+### IE 9 and 10
+
+We’ve been evaluating the cost of maintaining support for IE 9 and 10 and have determined that dropping them is the best way forward.
```
[the remark this answers is not in the document]
Comment (kara) [reply aio/content/guide/deprecations.md:662]: Looks like this comment still needs to be addressed?

**Review by kara — CHANGES_REQUESTED** [review]
Looks like some comments were missed (or maybe a bad rebase?)

File: aio/content/guide/deprecations.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (atscott) [reply aio/content/guide/deprecations.md]: Oops, looks like I forgot to save the file 🤦

File: aio/content/guide/deprecations.md [hunk 5]
```diff
@@ -562,7 +563,17 @@ In practical terms, the `package.json` of all `@angular` packages will change in
 
 For more information about the npm package format, see the [Angular Package Format spec](https://goo.gl/jB3GVv).
 
+{@a ie-9-10}
+### IE 9 and 10
```
Comment (kara) [inline aio/content/guide/deprecations.md:567]:
```suggestion
### IE 9 and 10 support
```

**Review by kara — APPROVED** [review]
LGTM, just one quick thing before merge

**Pull request closed without merging**